Update sustainability-design-methodology.md (#169)
MicrosoftDocs/well-architected: well-architected/sustainability/sustainability-design-methodology.md

- Usage over time (requests, users, API calls, etc.)
- Any extra measurement that is relevant to the application

> Further examples of implementing this can be found on the following pages:

Reviewer: ```suggestion
```
Unnecessary statement.

> - [DataOps - Measure Azure App Sustainability](https://learn.microsoft.com/en-us/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score)

Reviewer: ```suggestion
- To learn more, see [Measure Azure app sustainability by using the SCI score](/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score) in the Azure Architecture Center
```

- Usage over time (requests, users, API calls, etc.)
- Any extra measurement that is relevant to the application

Further examples of implementing this can be found on the following pages:

- [DataOps - Measure Azure App Sustainability](https://learn.microsoft.com/en-us/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score)

> - [ISE Developer Blog - Saving CO2 using location and time shifting in Azure](https://devblogs.microsoft.com/ise/saving-co2-using-location-and-time-shifting-in-azure/)

Reviewer: ```suggestion
```
Removing this suggestion, as the link is a bit outdated now and doesn't mention Azure carbon optimization, which gives you resource-insights to carbon emissions; something that blog post mentions isn't possible at the time of writing.

- Usage over time (requests, users, API calls, etc.)
- Any extra measurement that is relevant to the application

> To learn more, see [Measure Azure app sustainability by using the SCI score](/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score) in the Azure Architecture Center

Reviewer 2: ```suggestion
To learn more, see [Measure Azure app sustainability by using the SCI score](/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score) in the Azure Architecture Center.
```

> - To learn more, see [Measure Azure app sustainability by using the SCI score](/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score) in the Azure Architecture Center

Reviewer: ```suggestion
To learn more, see [Measure Azure app sustainability by using the SCI score](/azure/architecture/example-scenario/apps/measure-azure-app-sustainability-sci-score) in the Azure Architecture Center
```
Doesn't need to be a bulleted list.
